desktop/cmp/appbar/AppBar.scss

```diff
@@ -6,6 +6,10 @@
  */
 
 .xh-appbar {
+  .xh-app-menu {
+    margin-right: var(--xh-pad-px);
```

Comment:
Let's have this apply only when the button is actually on the left - otherwise we introduce some extra margin in the default position. You should be able to use the `:first-child` selector for that to get it done without needing to apply more classes in the component.